### PR TITLE
Upload DAP docs

### PR DESCRIPTION
**Pull request by hroncok**
For #325.

### CODE REVIEW

File: docs/developer_documentation/create_assistant/packaging_and_distributing.rst [hunk 1]
```diff
@@ -156,3 +157,59 @@ re-run `da tweak dap pack` and `da pkg lint foo-0.0.2.dap`, and upload it to
...
+  # assuming you dap is packaged in foo-0.0.1.dap
+  $ da extra dap upload -d foo-0.0.1.dap
+
+When you do this for the first time, you will be prompted for your authorization token.
+To obtain it, log into `DAPI <http://dapi.devassistant.org/>`_ with your Github
+or Fedora account, click on your username in the top menu and select *View profile*.
+Get the token on the bottom of your profile page.
+
+Note that by uploading the dap via command line, you agree to the `DAPI Terms of Use
+<https://dapi.devassistant.org/terms/>`_.
```
Comment (tradej) [inline docs/developer_documentation/create_assistant/packaging_and_distributing.rst]: I'm not sure if we're doing it, so please disregard this comment if we do: Would it be worth it to show this link to DAPI Terms in the CLI when uploading too? At least the first time the user uploads a DAP.

File: docs/developer_documentation/create_assistant/packaging_and_distributing.rst [hunk 2]
```diff
@@ -156,3 +157,59 @@ re-run `da tweak dap pack` and `da pkg lint foo-0.0.2.dap`, and upload it to
...
+
+Once provided the token, it is saved to ``devassistant.dapi.token`` git global configuration, so if
```
Comment (tradej) [inline docs/developer_documentation/create_assistant/packaging_and_distributing.rst]: Once you provide the token,....

...configuration. If...

**Comment by tradej** [comment]
Thanks for the docs. Except for the small edits on line 177, I think it's perfect and can go in.

File: docs/developer_documentation/create_assistant/packaging_and_distributing.rst (hunk 1 above)
Comment (hroncok) [inline docs/developer_documentation/create_assistant/packaging_and_distributing.rst]: https://github.com/devassistant/dap-dap/issues/5

**Comment by hroncok** [comment]
Changed accordingly.

File: docs/developer_documentation/create_assistant/packaging_and_distributing.rst (hunk 1 above)
Comment (tradej) [inline docs/developer_documentation/create_assistant/packaging_and_distributing.rst]: Okay, thanks.

**Comment by tradej** [comment]
Perfect, thanks.

**Pull request merged**